### ISSUE TITLE
APT repo broken

### ISSUE THREAD

**Issue by 0xpr03** [wireapp/wire-desktop#2344]
Wire version: 3.7.2891 Wire for web version: 2019.03.14.1105 Operating system: KUbuntu 18.03 Which antivirus software do you have installed: None

What steps will reproduce the problem?

1.  installed as documented for apt install
3.  did an apt update today

What is the expected result?
No errors during apt update fetching of the wire repo.
What is the actual result?
`N: Das Laden der konfigurierten Datei »main/binary-i386/Packages« wird übersprungen, da das Depot »https://wire-app.wire.com/linux/debian stable InRelease« die Architektur »i386« nicht unterstützt.`
Or when I directly browse to that link: AccessDenied

Please provide any additional information below. Attach a screenshot if possible.
Wire apt repos seem to be broken. Apt is unable to fetch anything.

**Comment by ffflorian** [comment]
Hi @0xpr03, are you using an 32-bit OS?

**Comment by ffflorian** [comment]
If you are using a 32-bit version, I have bad news: Electron [discontinued support for 32-bit Linux](https://electronjs.org/blog/linux-32bit-support).

If you are using KUbuntu on 64-bit, no need to worry. The repository is not broken, you just have to update your `wire-desktop.list` file to fetch only the 64-bit version. The new content of the file should be:

```
deb [arch=amd64] https://wire-app.wire.com/linux/debian stable main
```

---

**TL;DR:** Get rid of the error by running the following command in a terminal:
```
sudo sed -i -e 's/deb https/deb [arch=amd64] https/' "/etc/apt/sources.list.d/wire-desktop.list"
```

**Comment by 0xpr03** [comment]
Hi, indeed I've had `deb [arch=amd64] https://wire-app.wire.com/linux/debian stable main` in wire-desktop.list.
Added the 64bit tag and it worked again. Note: I've had just used your guide back then™ so there might be other people with this problem.

**Comment by ffflorian** [comment]
> I've had just used your guide back then

Thank you, I updated the Wiki already and will add the command, too.

**Comment by 0xpr03** [comment]
@ffflorian I'd add something to the wiki or check that "pages" is expanded per default because this is what you get at first:
![image](https://user-images.githubusercontent.com/1280209/54442565-232a5780-473f-11e9-9c30-532149aa62c1.png)

**Comment by ffflorian** [comment]
@0xpr03

Thanks for the hint, I added a table of contents.
